paymentsheet/src/test/java/com/stripe/android/link/FakeLinkProminenceFeatureProvider.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.link
+
+import com.stripe.android.link.domain.LinkProminenceFeatureProvider
+import com.stripe.android.paymentsheet.state.LinkState
+
+internal class FakeLinkProminenceFeatureProvider : LinkProminenceFeatureProvider {
```

Comment:
Let's make `LinkProminenceFeatureProvider` a functional interface so we can mock it with just `{ true }`.

Reply:
The idea is to add more Link prominence related checks to this class, but if this feels like solving a non-existent problem we can just have a `ShouldShowVerificationOnFlowControllerPaymentSelection` (or similar) usecase instead?